central/cve/image/datastore/datastore_impl.go

```diff
@@ -36,11 +37,12 @@ type datastoreImpl struct {
 	keyFence concurrency.KeyFence
 }
 
-func (ds *datastoreImpl) buildSuppressedCache() error {
+func (ds *datastoreImpl) buildSuppressedCache() {
 	query := pkgSearch.NewQueryBuilder().AddBools(pkgSearch.CVESuppressed, true).ProtoQuery()
 	suppressedCVEs, err := ds.searcher.SearchRawImageCVEs(accessAllCtx, query)
 	if err != nil {
-		return errors.Wrap(err, "searching suppress CVEs")
+		log.Error(errors.Wrap(err, "Vulnerability exception management may not function correctly. Failed to build cache of CVE exceptions."))
```

Comment:
@connorgorman how you feel about this? I think we shouldn't be blocking the Central due to the fact that exception management may not function correctly.

Reply:
Doesn't matter. It was logged one level up so the outcome is the same.